### PR TITLE
KAA-1576: Cassandra log appender - reviewed.

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: doc/Programming-guide/Key-platform-features/Data-collection/Cassandra-log-appender/index.md [hunk 1]
```diff
@@ -11,63 +10,121 @@ sort_idx: 10
...
+
+Use the checkboxes to make any field a partition and/or clustering key.
+
+Configure the key clustering in the **Clustering** section by adding column names and setting their order (**DESC** or **ASC**).
+
+>**NOTE:** You can specify Cassandra driver settings in the log appender configuration.
+>See [official documentation](http://docs.datastax.com/en/landing_page/doc/landing_page/current.html).
+{:.note}
+
+Below is an example configuration that matches the mentioned Avro schema.
```
Comment (Polinaa) [inline doc/Programming-guide/Key-platform-features/Data-collection/Cassandra-log-appender/index.md:127]: Look at the configuration below. Some fields and their values contain a needless space at the end. Error occurs while uploading  such configuration in Administration UI. Please remove these spaces.

Comment (hunch7) [reply doc/Programming-guide/Key-platform-features/Data-collection/Cassandra-log-appender/index.md:127]: Fixed.

File: doc/Programming-guide/Key-platform-features/Data-collection/Cassandra-log-appender/index.md [hunk 2]
```diff
@@ -87,29 +144,29 @@ An example configuration that matches to previously introduced Avro schema is as
     "tableNamePattern":"logs_$app_token_$config_hash",
     "columnMapping":[
         {
-            "type ":"HEADER_FIELD",
-            "value ":{
+            "type":"HEADER_FIELD",
+            "value":{
                 "string ":"applicationToken"
```
Comment (Polinaa) [inline doc/Programming-guide/Key-platform-features/Data-collection/Cassandra-log-appender/index.md]: Unnecessary space here. And in the "port" field above.

Comment (hunch7) [reply doc/Programming-guide/Key-platform-features/Data-collection/Cassandra-log-appender/index.md]: Fixed.

**Comment by Polinaa** [comment]
LGTM

**Review by vadimol — APPROVED** [review]

**Pull request merged**